### PR TITLE
Minor changes in documentation

### PR DESCRIPTION
**Pull request by geneticdrift**
- suppress warnings.
- improve readability by eliminating unnecessary hyphenation (changed from auto to manual and align to left instead of justify).

### CODE REVIEW

**Comment by MaxKellermann** [comment]
> Not sure if that's the right way to override just few styles

I don't know the right way either, but copying a large file from another project isn't acceptable. This adds a maintenance burden for keeping it in sync with upstream.

File: doc/user.rst [hunk 1]
```diff
@@ -717,10 +717,10 @@ MPD enables MixRamp if:
 - Cross-fade is enabled
 - :ref:`mixrampdelay <command_mixrampdelay>` is set to a positive
   value, e.g.::
-    mpc mixrampdelay 1
+  mpc mixrampdelay 1
```
Comment (MaxKellermann) [inline doc/user.rst:720]: This change breaks the code block

**Comment by MaxKellermann** [comment]
I cherry-picked a few commits to v0.23.x and master

**Comment by geneticdrift** [comment]
I made a new PR just for the custom css. Hopefully the right way this time.

File: doc/user.rst (hunk 1 above)
Comment (geneticdrift) [reply doc/user.rst:720]: Sorry.

**Pull request closed without merging**